lightkurve/interact.py

```diff
@@ -263,8 +266,18 @@ def stretch_change_callback(attr, old, new):
 
     return fig, stretch_slider
 
+def make_default_export_name(tpf, suffix='custom-lc'):
+    '''makes the default name to save a custom intetract mask'''
```

Comment:
Nitpick: pep8 includes a convention to always use double quote characters for triple-quoted strings/docstrings.